src/Microsoft.TemplateEngine.Cli/TemplateListResolutionResult.cs

```diff
@@ -258,6 +267,16 @@ public bool UsingContextMatches
             }
         }
 
+        // If BaseGetBestTemplateMatchList returned only partial matches with template name/lang, this is true.
+        // false otherwise.
+        public bool UsingPartialMatches
```

Comment:
minor: consider simplified format for such one statement cases in property getters...
```public bool UsingPartialMatches => _usingPartialMatches;```